### PR TITLE
Refine and finish bouncer implementation

### PR DESCRIPTION
**Pull request by bassosimone**

### CODE REVIEW

File: include/measurement_kit/ooni/bouncer.hpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+namespace bouncer {
+
+void post_net_tests(std::string base_bouncer_url, std::string test_name,
+                    std::string test_version, std::list<std::string> helpers,
+                    Callback<Error, Var<BouncerReply>> cb, Settings settings,
+                    Var<Reactor> reactor, Var<Logger> logger);
+
+} // namespace mk
+} // namespace ooni
+} // namespace bouncer
```
Comment (AntonioLangiu) [inline include/measurement_kit/ooni/bouncer.hpp]: I would first close the bouncer namespace, that is actually what happens but not what is written!

Comment (bassosimone) [inline include/measurement_kit/ooni/bouncer.hpp]: true! will fix it at godspeed

Comment (bassosimone) [inline include/measurement_kit/ooni/bouncer.hpp]: Fixed this everywhere I could find it

**Comment by AntonioLangiu** [comment]
I reviewed this PR and it looks good to me! Merging...

**Comment by AntonioLangiu** [comment]
I will open another PR to add documentation for the bouncer API.

**Pull request merged**